features/org.eclipse.pde.spies-feature/feature.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.eclipse.pde.spies"
+      label="%featureName"
+      version="1.0.0.qualifier"
```

Comment:
Alternatively the feature could be named `o.e.pde.spy` and its version could be aligned with the `o.e.pde` feature whose current version is `3.15.0`.